src/Umbraco.Web.BackOffice/Controllers/ContentController.cs

```diff
@@ -2435,6 +2435,7 @@ public IActionResult GetPublicAccess(int contentId)
                 .Select(_umbracoMapper.Map<MemberDisplay>)
                 .ToArray();
 
+            //TODO: change to role manager
```

Comment:
Yes public access will need to change to use the role manager. We have a separate task for public access